rdkit/Chem/EnumerateStereoisomers.py

```diff
@@ -354,4 +359,4 @@ def EnumerateStereoisomers(m, options=StereoEnumerationOptions(), verbose=False)
       if options.maxIsomers != 0 and numIsomers >= options.maxIsomers:
         break
     elif verbose:
-      print("%s    failed to embed" % (Chem.MolToSmiles(isomer, isomericSmiles=True)))
+      print(f"{Chem.MolToSmiles(isomer, isomericSmiles=True)}    failed to embed")
```

Comment:
`isomericSmiles=True` is the default. On the other hand, adding it here makes it clear that we want to see chirality information.

Reply:
As mentioned in the last comment, also since the file is targeted on Stereo-Isomers, we want to ensure that even if the default argument in `MolToSmiles()` is switched to False, we will always get `isomericSmiles=True`. Otherwise, it is quite hard to fix them